src/briefcase/platforms/macOS/__init__.py

```diff
@@ -912,8 +912,7 @@ def notarize(
         # Determine the arguments that would be needed to reproduce this notarization
         if installer_identity:
             identity_args = (
-                f"--identity {identity.id} "
-                f"--installer-identity {installer_identity.id}"
+                f"--identity {identity.id} --installer-identity {installer_identity.id}"
```

Comment:
This also got pushed into a single line - I can also disable concats like this one by disabling ruff momentarily if you'd prefer.